src/Cli/ConfigGenerator.cs

```diff
@@ -119,7 +119,12 @@ public static bool TryCreateRuntimeConfig(InitOptions options, FileSystemRuntime
                     // rest is not supported for cosmosdb_nosql yet.
                     if (!RestRuntimeOptions.DEFAULT_PATH.Equals(restPath))
                     {
-                        _logger.LogWarning("Configuration option --rest.path is not honored for cosmosdb_nosql since it does not support REST yet.");
+                        _logger.LogInformation("Configuration option --rest.path is not honored for cosmosdb_nosql since CosmosDB does not support REST.");
+                    }
+
+                    if (options.RestRequestBodyStrict is CliBool.False)
```

Comment:
nit: Is there a way to distinguish b/w user entering `--rest.request-body-strict true` vs not entering anything. Because, with Cosmos, using `--rest.request-body-strict` should lead to warning/error as it is a rest option
[This is a minor improvement and can be taken up subsequentially]

Reply:
When --rest.request-body-strict is not included in the command, a value of CliBool.None would be assigned.

Reply:
Yes, we can assigne a value of Cli.None when this flag is not present in the init command.